src/modules/iasql_functions/iasql.ts

```diff
@@ -1,7 +1,10 @@
 import * as levenshtein from 'fastest-levenshtein';
 import { default as cloneDeep } from 'lodash.clonedeep';
+import format from 'pg-format';
 import { Not, In, Between, LessThan, MoreThan, EntityMetadata } from 'typeorm';
 import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
+import { ColumnMetadata } from 'typeorm/metadata/ColumnMetadata';
+import { RelationMetadata } from 'typeorm/metadata/RelationMetadata';
```

Comment:
This does not work?

```suggestion
import { ColumnMetadata, RelationMetadata } from 'typeorm/metadata';
```

Reply:
had to revert this change, `typeorm/metadata` does not have the type declarations.